src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkSectionMultilineStringArgsInspection.kt

```diff
@@ -31,7 +31,9 @@ class SmkSectionMultilineStringArgsInspection : SnakemakeInspection() {
 
             val args = argumentList?.arguments ?: emptyArray()
             args.forEach { arg ->
-                if (checkRecursive(arg)) {
+                val visitor = SmkMultilineStringArgsInspectionVisitor()
```

Comment:
Why do we need a separate visitor object here per each section argument? Could we do args visitor stateless and use only a single instance per `SmkSectionMultilineStringArgsInspection` ? E.g. `SmkSectionMultilineStringArgsInspection` is stateless and and just registersProblem in given holder.

Reply:
One instance of visitor per inspection is enough, as it all use same SnakemakeBundle.message inside one ArgumentList. Now it reports about subexpression in which warning was detected, not whole argument (see test changes in commit bcffa17).

Reply:
I was wrong here, we need a psiElement in register problem, so we cannot use a single instance of stateless visitor per inspection. Let's leave current impl here.